src/components/ui/textarea.tsx

```
@@ -42,14 +42,22 @@ const Textarea = React.forwardRef<
         },
         ref
     ) => {
+        // Destructure to remove 'data-testid' from props, so it's not spread onto EditorContent
+        // The dataTestId variable (from destructuring the component's own props) is used for editorProps.attributes
+        const { 'data-testid': _forwardedDataTestId, ...restProps } = props;
         const editor = useEditor({
+            editorProps: {
+                attributes: {
+                    'data-testid': dataTestId ? `${dataTestId}-content` : 'tiptap-editor-content',
+                },
+            },
```

Biome (1.9.4)

[error] This property value named editorProps is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named editorProps


(lint/suspicious/noDuplicateObjectKeys)